pandaserver/taskbuffer/OraDBProxy.py

```diff
@@ -2281,7 +2281,7 @@ def updateJobStatus(self,pandaID,jobStatus,param,updateStateChange=False,attempt
                         _logger.debug("updateJobStatus : PandaID=%s skip to set holding since it is alredy in transferring" \
                                           % pandaID)
                         ret = 'alreadydone'
-                    elif oldJobStatus == 'holding' and jobStatus == 'holding' and (not 'jobDispatcherErrorDiag' in param or not param['jobDispatcherErrorDiag'] in [None,'']):
+                    elif oldJobStatus == 'holding' and jobStatus == 'holding' and ('jobDispatcherErrorDiag' not in param or not param['jobDispatcherErrorDiag'] in [None,'']):
```

Comment:
param['jobDispatcherErrorDiag'] not in [None,'']

Reply:
Very strange this was missed by my flake8...
Fixed.